flow/project.py

```diff
@@ -104,20 +104,63 @@
 
 class _condition(object):
 
-    def __init__(self, condition):
+    # All condition functions created using the _condition class are stored in this
+    # global registry variable, allowing direct equivalence comparison between
+    # condition objects.
+    _CONDITION_FUNCTIONS = dict()
```

Comment:
While this variable is technically global, I believe that it's convention to still write them lower-caps, because it is not on the module-level.

Reply:
Yes, I think you are correct.